app/Actions/CompileAssets.php

```diff
@@ -25,15 +25,15 @@ public function __invoke()
 
         $this->silentDevScript->add();
 
-        $this->logStep('Compiling project assets');
+        app('console-writer')->logStep('Compiling project assets');
```

Comment:
Could we instead:
- Create a `ConsoleWriter` class
- Inject that class into the `__invoke()` method
- Update all calls to our actions to happen via the IOC, which I *think* is something like this?

```php
app()->call(app(ConsoleWriter::class), '__invoke');
```

Reply:
(If we did that, we may want to make a helper that was something like `makeAndInvoke(ConsoleWriter::class)` or something

Reply:
Done.